examples/Bunch/README.rst

```diff
@@ -0,0 +1,35 @@
+Bunch: Thomson scattering from laser electron-bunch interaction
+===============================================================
+
+* author:      Richard Pausch <r.pausch (at) hzdr.de>, Rene Widera <r.widera (at) hzdr.de>
+* maintainer:  Richard Pausch <r.pausch (at) hzdr.de>
+
+This is a simulation of an electron bunch that collides head-on with a laser pulse.
+Depending on the number of electrons in the bunch, their momentum and their distribution and depending on the laser wavelength and intensity, the emitted radiation differs.
```

Comment:
I know :wink: ... I counted on your self-aware freedom to put a full stop there somewhere and suggested an alternative way of modifying the long sentence (in case you wanted to keep it long) that makes you not read twice to get it.